deploy/addons/registry-aliases/node-etc-hosts-update.tmpl

```diff
@@ -29,7 +29,7 @@ spec:
                   name: registry-aliases
                   key: registryAliases
           command:
-            - bash
+            - ash
```

Comment:
Could consider using standard /bin/sh here

Reply:
@tsraza I agree with @afbjorklund

Reply:
changed to `sh` and tested locally in virtualbox -- it works